dependencies/Makefile.linux

```diff
@@ -15,11 +15,12 @@ PICO_PACKAGE = libpico.tar.bz2
 LUA_PACKAGE = lua-5.2.3.tar.gz
 OPENSSL_PACKAGE = libssl_1.1.tar.xz
 
-PACKAGES = qt open-al ois pico lua-gd lua assimp open-ssl
+PACKAGES = qt open-al ois pico lua-gd lua assimp
 PACKAGES_CLEAN = $(addsuffix -clean, $(PACKAGES))
 
 ifeq ($(UBUNTU_VERSION), 22.04)
 	PACKAGES += open-ssl
+	PACKAGES_CLEAN += open-ssl-clean
```

Comment:
Is this really needed?
Makefile uses lazy evaluation.
I tested locally and without this works fine, otherwise the target is added twice.

Reply:
You are right, this is not needed with lazy evaluation.